config/test_map_config.kv

```diff
@@ -0,0 +1,105 @@
+"menus" 
```

Comment:
Having this as one kv file will not work very well for the workshop, but since workshop is not implemented we can revisit this later

Reply:
the way I see it being done, is that this file is static, at all times, it's literally just for the 3 (4 if paintgun gets added to this list) campaigns, workshops would be loaded from different files (probably files brought into the UI's view via a file that workshop mods ship with?) and is simply appended afterwards, not just assuring that the campaigns are appended first, but also allowing us to easily make changes to the main campaign KV file.
wouldn't even surprise me if you guys hardcoded this file so it cannot be tempered with by sourcemods.
it's purely there for the P2CE/P2 side of campaigns.